packages/components/table/src/filter-panel.vue

```diff
@@ -69,7 +71,10 @@
     <template #default>
       <span
         v-click-outside:[popperPaneRef]="hideFilterPanel"
-        class="el-table__column-filter-trigger el-none-outline"
+        :class="[
+          `${ns.namespace.value}-table__column-filter-trigger`,
```

Comment:
I don't think we really need this `.value` here since in template it automatically unwraps the refs

Reply:
> I don't think we really need this `.value` here since in template it automatically unwraps the refs

emmm,  vue not automatically unwraps 

![image](https://user-images.githubusercontent.com/26672484/150621241-bd13002e-683b-4912-b176-5e32dc73cdb4.png)